cmd/helm-operator/main.go

```diff
@@ -159,6 +160,10 @@ func main() {
 		logger = log.With(logger, "caller", log.DefaultCaller)
 	}
 
+	// configure go logger to output using go-kit log
+	logWriter := helm.NewLogWriter(logger)
```

Comment:
Given this is now utilized as a global thing, I think it should be moved to an `utils` package.

Reply:
@hiddeco done